Move phone, email, software OATH authentication methods from beta to v1.0 (#16840)
microsoftgraph/microsoft-graph-docs: api-reference/v1.0/api/phoneauthenticationmethod-disablesmssignin.md

---
title: "phoneAuthenticationMethod: disableSmsSignIn"

> description: "Disable SMS sign-in for a mobile phone"

Reviewer: ```suggestion
description: "Disable SMS sign-in for a mobile phone registered to a user."
```

ms.localizationpriority: medium
author: "mmcla"
ms.prod: "identity-and-sign-in"
doc_type: "apiPageType"
---

# phoneAuthenticationMethod: disableSmsSignIn

Namespace: microsoft.graph

> Disable SMS sign-in for an existing `mobile` phone number. 
> 
> >**Note:** The number will no longer be available for SMS sign-in, which can prevent your user from signing in.

Reviewer: ```suggestion
Disable SMS sign-in for an existing `mobile` phone number registered to a user. The number will no longer be available for SMS sign-in, which can prevent your user from signing in.
```

|:---------------------------------------|:-------------------------|
| Delegated (work or school account)     | UserAuthenticationMethod.ReadWrite.All |
| Delegated (personal Microsoft account) | Not supported. |
| Application                            | UserAuthenticationMethod.ReadWrite.All |

For delegated scenarios where an admin is acting on another user, the admin needs one of the following [Azure AD roles](/azure/active-directory/users-groups-roles/directory-assign-admin-roles#available-roles):
* Global administrator
* Privileged authentication administrator
* Authentication administrator

## HTTP request

<!-- { "blockType": "ignored" } -->

```http
POST /me/authentication/phoneMethods/{id}/disableSmsSignIn
POST /users/{id | userPrincipalName}/authentication/phoneMethods/{id}/disableSmsSignIn
```
The value of `id` for the `mobile` phoneType is `3179e48a-750b-4051-897c-87b9720928f7`.

## Request headers

| Name          | Description   |
|:--------------|:--------------|
| Authorization | Bearer {token}. Required. |

## Request body

Do not supply a request body for this method.

## Response

> If successful, this method returns a `200 OK` response code. It does not return anything in the response body.

Reviewer: Hope this is correct?
```suggestion
If successful, this method returns a `204 No Content` response code. It does not return anything in the response body.
```

## Examples

The following example shows how to call this API.

### Request

The following is an example of the request.

> # [HTTP](#tab/http)

Reviewer: ```suggestion
```

<!-- {
  "blockType": "request",
  "name": "phoneauthenticationmethod_disablesmssignin"
}-->

```http
POST https://graph.microsoft.com/v1.0/me/authentication/phoneMethods/3179e48a-750b-4051-897c-87b9720928f7/disableSmsSignIn
```

---

### Response

The following is an example of the response.
<!-- {
  "blockType": "response"
} -->

```http

> HTTP/1.1 200 OK

Reviewer: ```suggestion
HTTP/1.1 204 No Content
```
